contrib/linux/sensors/file_watch_sensor.py

```diff
@@ -44,8 +44,8 @@ def cleanup(self):
 
             try:
                 self._tail.notifier.stop()
-            except Exception:
-                pass
+            except Exception as e:
+                self._logger.exception("Unable to stop the tail notifier : '%s'", e)
```

Comment:
Since you're using logger.exception, you don't need to pass e in the text message.